### PR TITLE
Subnav white background adjustment

### LINKED ISSUE #18863: Navigation block mobile styling on front
Currently, the sub nav menu isn't so great on mobile, neither is default style. It would be great to have exploration in what this could be, without being too opinionated.

### LINKED ISSUE #18969: Default subnav menu doesn't show on white background
Right now the following shows by default.

2019-12-06 at 11 40

### PR DESCRIPTION
**Pull request by karmatosed**
Brings in off-white color to make it work on white backgrounds, for example Twenty Twenty. Fixes #18969

image

### CODE REVIEW

**Comment by shaunandrews** [comment]
I don't mind the off-white — I don't love it either. Perhaps adding a border instead would help, and likely be a better solution than a background color.

**Comment by MichaelArestad** [comment]
> I don't mind the off-white — I don't love it either. Perhaps adding a border instead would help, and likely be a better solution than a background color.

I agree. I'd lean toward a border myself.

**Comment by melchoyce** [comment]
Agreed — could we style it like our existing dropdown menus, like the block library or settings dropdown?

**Comment by karmatosed** [comment]
For now, I have styled as per the editor but I do think there's a fair point that it should have different styling as it is a different element. Also of note, the concept of having colour as the focus was good to explore over an explicit border. I have also to be consistent brought a border into the dark version.

**Comment by karmatosed** [comment]
New screenshots:

image


image

**Comment by shaunandrews** [comment]
For the dark version, it might make sense to use a lighter border. So for the light version, there's a dark border (and I'd even suggest a little darker than what you have) and for the dark version, there's a light border. This helps ensure that the menu is noticeable across various color schemes.

File: packages/block-library/src/navigation/style.scss [hunk 1]
```diff
@@ -60,10 +60,11 @@
 		// Sub-menus Flyout
 		& > li > ul {
 			background: #fff;
-			-webkit-box-shadow: -2px 4px 4px -1px rgba(97, 97, 97, 0.153);
-			-moz-box-shadow: -2px 4px 4px -1px rgba(97, 97, 97, 0.153);
-			border-radius: 4px;
-			box-shadow: -2px 4px 4px -1px rgba(97, 97, 97, 0.153);
+			-webkit-box-shadow: 0 3px 30px rgba(25, 30, 35, 0.1);
```
Comment (shaunandrews) [inline packages/block-library/src/navigation/style.scss]: Is there a reason you're not using straight black (`rgba( 0, 0, 0, x)`) or white ( `rgba( 255, 255, 255, x )` )?

**Comment by karmatosed** [comment]
@shaunandrews that darkness on the light is directly from the existing menu in the editor so I think keeping it as that makes sense for now. Same with the darker, I think to keep it dark works, rather than reversing the colourways.

File: packages/block-library/src/navigation/style.scss (hunk 1 above)
Comment (karmatosed) [reply packages/block-library/src/navigation/style.scss]: This is directly from existing shadows in Gutenberg so pulling from those.

**Comment by karmatosed** [comment]
Added an iteration on dark mode for @shaunandrews  🙌

**Pull request merged**